lib/Raven/Client.php

```diff
@@ -881,9 +1016,12 @@ public function capture($data, $stack = null, $vars = null)
             }
 
             if (!isset($data['stacktrace']) && !isset($data['exception'])) {
-                $data['stacktrace'] = array(
-                    'frames' => Stacktrace::fromBacktrace($this, $stack, isset($stack['file']) ? $stack['file'] : __FILE__, isset($stack['line']) ? $stack['line'] : __LINE__)->getFrames(),
-                );
+                $data['stacktrace'] = [
+                    'frames' => Stacktrace::fromBacktrace(
+                        $this, $stack, isset($stack['file']) ? $stack['file'] : __FILE__,
+                        isset($stack['line']) ? $stack['line'] : __LINE__
```

Comment:
>Lines should not to be longer than 120 characters.

I absolutely agree on the topic, although 120 characters is maybe a common setting or something you decided yourself. By the way, I'm not interested in starting a discussion for a so small change, I don't think it's useful at all to discuss on every change you do, so if you evaluated all the pros/cons of this change and still think that it's worth although it "breaks" the colorization of the correct line in Sentry, use `__LINE__ - 2` and I will be ok with that